custom/icds_reports/reports/daily_indicators.py

```diff
@@ -0,0 +1,86 @@
+from custom.icds_reports.models import AggAwcDailyView
+from dateutil.relativedelta import relativedelta
```

Comment:
nit: I think it's better to use `timedelta` instead of `relativedelta` when possible. Two reasons:

1) `timedelta` is part of the standard library so it theoretically removes an external dependency (not that we could remove `dateutil` anytime soon)
2) `relativedelta` has arguments both `day` and `days` (similar for other time frames), one of which performs the arithmetic, the other replaces the value. We've had bugs come up before where these were switched unintentionally and its super hard to track down